tokio/tests/process_arg0.rs

```diff
@@ -0,0 +1,14 @@
+#![warn(rust_2018_idioms)]
+#![cfg(feature = "full")]
+
+use tokio::process::Command;
```

Comment:
CI fails on windows due to a warning that this import is unused, which it is on windows.